api/server/image.go

```diff
@@ -112,7 +113,7 @@ func (s *Server) postImagesCreate(ctx context.Context, w http.ResponseWriter, r
 			OutStream:   output,
 		}
 
-		err = s.daemon.Repositories().Pull(image, tag, imagePullConfig)
+		err = s.daemon.Repositories().Pull(image, tag, imagePullConfig, dryRun != "")
```

Comment:
The CLI won't send it, but I could see someone using the API directly inserting `dryRun=false`.

...which reminds me that this parameter should be documented in docs/reference/api/docker_remote_api.md.